TCMSApp/src/client/app/admin/admin.route.js

```diff
@@ -25,7 +25,7 @@
                 config: {
                     url: '/team',
                     templateUrl: 'app/admin/team.html',
-                    controller: 'AdminController',
+                    controller: 'TeamController',
                     controllerAs: 'vm',
```

Comment:
maybe something like teamVm would be better. 
Why? So that we can access parent controller by its name from any child controller